mock_ros2_project/service_server.py

```
@@ -38,18 +52,22 @@ def add_two_ints_callback(self, request, response):
         return response
 
 
-def main(args=None):
-    """Main function to run the service server."""
+def main(args=None) -> None:
+    """Main function to run the service server.
+    
+    Args:
+        args: Command line arguments (optional)
+    """
     rclpy.init(args=args)
     
-    service_server = ServiceServer()
+    service_server_node = ServiceServer()
     
     try:
-        rclpy.spin(service_server)
+        rclpy.spin(service_server_node)
     except KeyboardInterrupt:
         pass
```

Silently swallowing the KeyboardInterrupt makes it harder to diagnose user-initiated shutdowns. Consider logging a message (e.g., `self.get_logger().info('Service server interrupted, shutting down…')`) so operators can see why the node stopped.